checkstyle/checkstyle.xml

```diff
@@ -298,7 +298,12 @@
             <property name="max" value="12"/>
         </module>
         <module name="NPathComplexity">
-            <property name="max" value="50"/>
+            <!-- TODO Agree on the value
+                 Checkstyle 7.7 improved the NPath algorithm further that leads to higher NPath counts.
+                 7.6 passes with max 50, 7.7 needs it to be set to at least 216
+                 the default threshold is 200
+            -->
+            <property name="max" value="250"/>
```

Comment:
Let's set it to a value that passes, and later we'll revisit it

And please remove that comment from the code

Reply:
done